packages/http/src/mocker/__tests__/functional.spec.ts

```diff
@@ -180,16 +177,18 @@ describe('http mocker', () => {
 
       describe('the media type requested does not match the example', () => {
         test('throw exception', () => {
-          const rejection = mocker.mock({
-            resource: httpOperations[0],
-            input: Object.assign({}, httpRequests[0], {
-              data: Object.assign({}, httpRequests[0].data, {
-                headers: { 'Content-type': 'application/yaml' },
+          return expect(
+            mocker.mock({
+              resource: httpOperations[0],
+              input: Object.assign({}, httpRequests[0], {
+                data: Object.assign({}, httpRequests[0].data, {
+                  headers: { 'Content-type': 'application/yaml' },
+                }),
               }),
-            }),
+            })
+          ).resolves.toMatchObject({
```

Comment:
I'd lie to see we match against the body too. It's equally important i think

Reply:
Done in d6dc96f